src/NServiceBus.Metrics.PerformanceCounters.Tests/PerformanceCounterUpdaterTests.cs

```diff
@@ -100,13 +104,13 @@ public void Duration_probes_within_payload_should_be_converted_into_performance_
             // asserting the number of accessed counters
             Assert.AreEqual(3 + 3, cache.Count);
 
-            var counter1 = cache.Get(new CounterInstanceName("Critical Time", "Sender@af016c07"));
-            var counter1average = cache.Get(new CounterInstanceName("Critical Time Average", "Sender@af016c07"));
-            var counter1averageBase = cache.Get(new CounterInstanceName("Critical Time AverageBase", "Sender@af016c07"));
+            var counter1 = cache.Get(new CounterInstanceName(criticalTimeProbe.Name, "Sender@af016c07"));
```

Comment:
I think we want to leave these counters names as they should be. Otherwise, how do we test `GetAverageTimerCounterName`, `GetAverageTimerBaseCounterName` ? Or do we say that this is API approvals' job?